proto/proto/transaction.proto

```diff
@@ -52,6 +52,10 @@ message Action {
     stake.ValidatorDefinition validator_definition = 16;
     ibc.IBCAction ibc_action = 17;
 
+    /*
+    ICS20Withdrawal withdrawal = 18;
+    */
```

Comment:
We should add in the action now, and reject it until the implementation is ready.  Otherwise we don't have a good way to incrementally implement more and more functionality, since the datatypes aren't there yet.